libs/blocks/src/lib/hero/content-less/index.tsx

```diff
@@ -9,10 +8,14 @@ const ContentLess = ({
   children,
 }: HeroBlockProps) => {
   return (
-    <BlockWrapper className={className} background="light">
-      <Content title={title} description={description} center={true}>
-        {children}
-      </Content>
+    <BlockWrapper
+      title={title}
+      description={description}
+      heading="H1"
+      className={className}
+      background="light"
```

Comment:
Is heading always going to be `H1` and background `light`?

Reply:
For background i have added the prop but for H1 yes it should always be H1 because its hero block.